polaris-service/src/test/java/io/polaris/service/admin/PolarisServiceImplIntegrationTest.java

```diff
@@ -137,11 +135,11 @@ public void tearDown() {
                       .equals(PolarisEntityConstants.getNameOfPrincipalServiceAdminRole()))
           .forEach(
               principalRole -> {
-                try (Response innerResponse =
```

Comment:
Same as the other note on try-finally

Reply:
same answer as above for try-finally with empty try block